apollo-sample/build.gradle

```diff
@@ -29,6 +29,7 @@ dependencies {
   compile project(':apollo-converters:moshi')
   compile project(':apollo-converters:pojo')
   compile project(":apollo-runtime")
+  //TODO: remove dependency after new snapshot release of the plugin
```

Comment:
I would actually have loved the comment there :), would have saved me e bit of time